joomla/joomla-cms
# Correct jQuery submit trigger on form submit

PR #3484 introduced the ability to bind jquery functionality on submit form event. The problem is that this doesn’t fire as expected as observed on #6207.
#### What is changed?

Instead of submit

``` javascript
        jQuery(form).submit();
```

it should be:

``` javascript
        jQuery(form).trigger("submit");
```
#### Test instructions:

Apply patch
Open /administrator/components/com_content/views/article/tmpl/edit.php
and insert

``` php
JFactory::getDocument()->addScriptDeclaration('
jQuery(document).ready(function() {
    jQuery("#item-form").bind("submit", function() {
        console.log("works");
    });
});
');
```

just before ? > at around line 75

Try to create (save) an article, you should get in browsers log `works`

OR follow [these](https://github.com/joomla/joomla-cms/issues/6207#issuecomment-86039978) instructions
